.travis.yml

```diff
@@ -1,16 +1,48 @@
+# https://docs.travis-ci.com/user/languages/python/#python-37-and-higher
+dist: xenial
 language: python
 python:
-  - '2.7'
-  - '3.6'
+  - '3.7'
```

Comment:
Mike its not possible to support both py36 and py37? Most of contributors still using fedora < `29`.

Reply:
Possible, but async and await became reserved keywords in 3.7, which is one of the main reasons I was targeting 3.7 instead of 3.6. So that forward-moving use of async/await is consistent from keyword, usability, and performance standpoints.